game.hpp

```diff
@@ -30,12 +40,12 @@ void delete_old_position(short x, short y) {
 }
 
 bool check_obstacles(short x, short y) {
-    return map[y][x] == '#' || map[y][x] == '|';
+    return map[y][x] == '-' || map[y][x] == '|';
```

Comment:
after this change from '#' to '|' still can detect the obstacles?

Reply:
Oh damn.. I actually just wanna try out if using `-` would make the map looks less complicated.

Reply:
That will make the map weird bcz the player cannot run to the extra space, make confusion?